### DISCUSSION TITLE
Packages

### DISCUSSION THREAD

**Discussion by mukeenanyafiq** [termux/termux-packages#7904] [Q&A] [answered]
How do I install a packages into Termux?

**Comment by thunder-coding** [comment, marked as answer]
To manage packages in Termux, you need to use the `pkg` or `apt` utility. It is recommended to use `pkg` if you are learning about Linux and related and aren't good at using `apt`.

- Installing packages
```
pkg install <package name>
```

- Uninstalling packages
```
pkg uninstall <package name>
```
- Updating packages
```
pkg up
```

For more information see https://wiki.termux.com/wiki/Package_Management
